articles/user-guide/basics.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Q# Basics
+description: Basic concepts of Q#
+author: gillenhaalb
+ms.author: a-gibec@microsoft.com
+ms.date: 02/28/2020
+ms.topic: article
+uid: microsoft.quantum.guide.basics
+---
+
+# Q# Basics
+
+In this section we present a brief introduction to the basic building blocks of Q#.
+
+For a quick overview of what Q# is and where it fits in as a fundamental component of the Quantum Development Kit, you can check out [What is Q#?](xref:microsoft.quantum.overview.qsharp). 
+
+## What is a quantum program?
+
+From a technical perspective, a quantum program can be seen as a particular set of classical subroutines which, when called, perform certain operations on a quantum system.
+An important consequence of that view is that a program written in Q# does not directly model qubits themselves, but rather describes how a classical control computer interacts with those qubits.
+By design, Q# thus does not define quantum states or other properties of quantum mechanics directly.
+For instance, consider the state $\ket{+} = \left(\ket{0} + \ket{1}\right) / \sqrt{2}$ discussed in the [Quantum Computing Concepts](xref:microsoft.quantum.concepts.intro) guide.
+To prepare this state in Q#, we use the facts that the qubits are initialized in the $\ket{0}$ state, and that $\ket{+} = H\ket{0}$, where $H$ is the Hadamard transform, implemented by the [`H` operation](](xref:microsoft.quantum.intrinsic.h):
+
+```qsharp
+using (qubit = Qubit()) {
+    // At this point, qubit is in the state |0⟩.
+    H(qubit);
+    // We've now applied H, such that our qubit is in H|0⟩ = |+⟩, as we wanted.
+}
+```
+
+## Quantum states in Q#
+
+Importantly, in writing the above program, we did not explicitly refer to the state within Q#, but rather described how the state was *transformed* by our program.
+Thus, similar to how a graphics shader program accumulates a description of transformations to each vertex, a quantum program in Q# accumulates transformations to quantum states.
```

Comment:
😆 I thought the exact same thing. Left it in because I wasn't sure if it was a new addition that I'd brought over from the first iteration you and Chris worked on, but absolutely agree. Would scare off twenty people for every one person who it helps.